pymc3/distributions/continuous.py

```diff
@@ -19,8 +22,8 @@
 from .special import log_i0
 from ..math import invlogit, logit, logdiffexp
 from .dist_math import (
-    bound, logpow, gammaln, betaln, std_cdf, alltrue_elemwise,
-    SplineWrapper, i0e, normal_lcdf, normal_lccdf
+    alltrue_elemwise, betaln, bound, gammaln, i0e, logpow, normal_lccdf,
+    normal_lcdf, normallogcdf, SplineWrapper, std_cdf, zvalue,
```

Comment:
Ah, I missed the addition of another normal log CDF.
It would be better to remove the redundancy before merging into master.